gradle/versions.gradle

```diff
@@ -39,27 +39,27 @@ dependencyManagement {
 
     dependency 'com.splunk.logging:splunk-library-javalogging:1.9.0'
 
-    dependency 'com.squareup.okhttp3:okhttp:4.9.1'
+    dependency 'com.squareup.okhttp3:okhttp:4.9.3'
 
-    dependency 'commons-codec:commons-codec:1.13'
+    dependency 'commons-codec:commons-codec:1.15'
 
     dependency 'commons-io:commons-io:2.11.0'
 
     dependency 'dnsjava:dnsjava:3.4.0'
 
     dependency 'info.picocli:picocli:4.6.1'
 
-    dependency 'io.grpc:grpc-core:1.39.0'
-    dependency 'io.grpc:grpc-netty:1.39.0'
-    dependency 'io.grpc:grpc-stub:1.39.0'
+    dependency 'io.grpc:grpc-core:1.37.1'
```

Comment:
I'd like exposition as to why grpc is going back two minor versions.  Also, does this have the same security baseline as 1.39.0?  i.e. are we going back on CVE exposures by doing this?

Reply:
agree. changing this back to 1.39.0 and asking customer for their scan report